src/main/java/com/minecolonies/util/PermissionUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ *
+ */
+public final class PermissionUtils {
+
+    public static List<Permissions.Player> getPlayersWithAtleastRank(Colony colony, Permissions.Rank rank) {
+        List<Permissions.Player> playersWithAtLeastRank = new ArrayList<Permissions.Player>();
+
+        final Permissions permissions = colony.getPermissions();
+        final Map<UUID, Permissions.Player> players = permissions.getPlayers();
+
+        for (Permissions.Player player: players.values()) {
+            if (player.getRank().ordinal() <= rank.ordinal()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)